auth_token/utils.py

```diff
@@ -174,3 +177,10 @@ def takeover(request, user):
         request.token.user_takeovers.update(is_active=False)
         request.token.user_takeovers.create(user=user, is_active=True)
         return True
+
+
+def generate_key(length=20):
+    """
+    Random ID generating of 'length' bytes
+    """
+    return force_text(binascii.hexlify(os.urandom(length)))
```

Comment:
stačí používat str místo force_text (force kvůli pythonu 2.7)

Reply:
I've tried that but 9 old tests are failing with that. I propose fix it later as it's not my change (I've just  copied the old code to another file)